book/src/pong/03_sprites.md

```diff
@@ -83,15 +83,15 @@ If you run this you should now see the ball for this pong game somewhere in the
 
 The GBA renders to the screen one pixel at a time a line at a time from left to right.
 After it has finished rendering to each pixel of the screen, it briefly pauses rendering before starting again.
-This period of no drawing is called `vblank`, which stands for the 'vertical blanking interval'.
+This period of no drawing is called the 'vertical blanking interval' which is shortened to `vblank`.
 There is also a 'horizontal blanking interval', but that is outside of the scope of this book.
```

Comment:
This should therefore also be changed to 'tutorial'

Reply:
Yeah, definitely